package.json

```diff
@@ -8,20 +8,24 @@
   "source": "ts/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.mjs",
-  "unpkg": "dist/index.umd.js",
+  "browser": "dist/browser/schema.mjs",
+  "unpkg": "dist/browser/schema.umd.js",
```

Comment:
Even though the [`browser` field can be a map](https://github.com/defunctzombie/package-browser-field-spec#replace-specific-files---advanced) to handle both modern/module/`mjs` file and the UMD file, it caused issues when used from StencilJS.

The current approach assumes that the `browser` field is used from other project and will go through their own compilation process, and the `unpkg` field will get served through CDN usage and not require any post-processing.